src/main/java/com/aws/greengrass/smbridge/TopicMapping.java

```diff
@@ -15,8 +15,8 @@
 import lombok.NonNull;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
+import java.lang.reflect.Array;
+import java.util.*;
```

Comment:
Remove .*; and use package name instead. There is also a setting in IntelliJ to avoid this in future,

Reply:
Should be all set now. Thanks for catching that! I also cleaned up the other import lines that I didn't need any more...